messages/de.json

```diff
@@ -0,0 +1,1393 @@
+        "vigilante": ["Mitglied der Bürgerwehr", "Mitglierder der Bürgerwehr"],
+        "villager": ["Bürger", "Bürger"],
+        "village drunk": ["Besoffener Bürger", "Besoffener Bürger"],
+        "warlock": ["Zauberer", "Zauberer"],
+        "werecrow": ["Wehrkrähe", "Wehrkrähe"],
+        "werekitten": ["Wehrkatze", "Wehrkatze"],
+        "wild child": ["Wildes Kind", "Wilde Kinder"],
+        "wolf": ["Wolf", "Wölfe"],
+        "wolf cub": ["Wolfsjunge", "Wolfsjungen"],
+        "wolf gunner": ["Wolf Kanonier", "Wolf Kanoniere"],
+        "wolf mystic": ["Wolf Mystiker", "Wolf Mystiker"],
+        "wolf shaman": ["Wolf Shamane", "Wolf Shamanen"],
+        "**": "The following items are names for the 'team reveal' role reveal option",
+        "village member": ["Dorfmitglied", "Dorfmitglieder"],
+        "neutral player": ["Neutraler Spieler", "Neutraler Spieler"],
+        "wolfteam player": ["Wolfteam Spieler", "Wolfteam Spieler"],
+        "***": "The following items are special keys used in player stats",
+        "lover": ["Liebhaber", "Liebhaber"],
+        "vg activated": ["vg activiert", "vgs activiert"],
+        "vg driven off": ["vg abgefahren", "vgs abgefahren"],
+        "entranced": ["verzaubert", "verzaubert"],
+        "****": "The following items are old roles that are no longer in use, but need to be here for legacy reasons",
+        "bureaucrat": ["Bürokrat", "Bürokraten"]
+    },
+    "_gamemodes": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Gamemodes",
+        "aleatoire": "zufällig",
+        "alpha": "Alpha",
+        "boreal": "Nördlich",
+        "charming": "charmant",
+        "classic": "Klassisch",
+        "default": "Standard",
+        "drunkfire": "betrunkenes Feuer",
+        "evilvillage": "böses Dorf",
+        "foolish": "töricht",
+        "guardian": "Wächter",
+        "lycan": "Lykianer",
+        "mad": "Wütend",
+        "maelstrom": "Sog",
+        "mudkip": "Mudkip",
+        "noreveal": "Noreveal",
+        "random": "zufällig",
+        "rapidfire": "Schnellfeuer",
+        "roles": "Rollen",
+        "sleepy": "Schläfrig",
+        "valentines": "Valentinstag",
+        "**": "The following items are old modes that are no longer in use, but need to be here for legacy reasons",
+        "villagergame": "villagergame"
+    },
+    "_role_aliases": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Role_aliases",
+        "crazed shaman": ["cs"],
+        "guardian angel": ["ga"],
+        "mad scientist": ["ms"],
+        "matchmaker": ["mm"],
+        "vengeful ghost": ["vg"],
+        "village drunk": ["drunk"]
+    },
+    "_commands": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Commands",
+        "": [""],
+        "abstain": ["abstain", "abs", "nolynch", "nl", "novote", "nv"],
```

Comment:
I see none of the commands have been translated yet. What I recommend doing is putting 1 (or more) translated versions of the command first, and then the original English command as the very last one in the list. That way people reading our online (English) documentation can still use the commands and they work as expected.

You'll see a lot of commands starting with "f" like "ftotem" or "fjoin" -- the "f" is short for "force" in this context ("force totem", "force join").

Reply:
I didn't translate the commands because I didn't want to risk the bot crashing because they are in the source code.